README.md

```diff
@@ -2397,6 +2397,45 @@ Other Style Guides
       'Batman',
       'Superman',
     ];
+    
+    // bad
+    function createHero(
+      firstName,
+      lastName,
+      inventorOf
+    ) {
+      // does nothing
+    }
+    
+    // good
+    function createHero(
+      firstName,
+      lastName,
+      inventorOf,
+    ) {
+      // does nothing
+    }
+    
+    // good
+    function createHero(firstName,
+                        lastName,
```

Comment:
Specifically, "good" doesn't mean "passes the linter rule", "good" means "conforms to our styleguide"